P5/Source/Specs/att.measurement.xml

```diff
@@ -46,7 +46,8 @@ $Id$
           <gloss versionDate="2007-05-02" xml:lang="zh-TW">公尺</gloss>
           <gloss versionDate="2007-06-12" xml:lang="fr">mètre</gloss>
           <gloss versionDate="2007-11-06" xml:lang="it">metro</gloss>
-          <gloss versionDate="2007-05-04" xml:lang="es">metro</gloss>
+          <gloss versionDate="2022-05-12" xml:lang="ja">metro</gloss>
+	  <gloss versionDate="2007-11-06" xml:lang="it">メートル</gloss>
```

Comment:
probably a copy paste error here?! duplicate Italian gloss and the Japanese gloss in Italian?

Reply:
Dear Peter, thank you for your cheking. I've deleted the duplication.